apps/heureka/src/components/services/ComponentInstancesList.test.jsx

```diff
@@ -0,0 +1,134 @@
+/*
+ * SPDX-FileCopyrightText: 2024 SAP SE or an SAP affiliate company and Greenhouse contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import React from "react"
+import { render, screen, act, fireEvent } from "@testing-library/react"
+import { vi } from "vitest"
+import {
+  StoreProvider,
+  useGlobalsQueryClientFnReady,
+  useGlobalsQueryOptions,
+  useGlobalsActions,
+} from "../StoreProvider"
+import { useQuery } from "@tanstack/react-query"
+import ComponentInstancesList from "./ComponentInstancesList"
+
+// Mock dependencies
+vi.mock("@tanstack/react-query", () => ({
+  useQuery: vi.fn(),
+}))
+
+vi.mock("../StoreProvider", () => ({
+  // Mock StoreProvider to just render children
+  StoreProvider: ({ children }) => <div>{children}</div>,
+  useGlobalsQueryClientFnReady: vi.fn(),
+  useGlobalsQueryOptions: vi.fn(),
+  useGlobalsActions: vi.fn(),
+}))
+
+vi.mock("../shared/LoadElement", () => ({
+  default: () => <div>Loading...</div>, // Mock loading element
+}))
+
+vi.mock("../shared/Helper", () => ({
```

Comment:
Why you also mock it this way?

Reply:
Cause I did not want to test the severityString, highestSeverity functions from helper inside this component. There is another test for helper.